Service/src/main/java/Users/Client.java

```diff
@@ -0,0 +1,20 @@
+package Users;
+
+import TIcket.Ticket;
+import org.example.SetClassId;
+
+public class Client extends User {
+    private static int classId;
+    private static String role = "CLIENT";
+
+    //Used as a stub method
+    public void getTicket(Ticket ticket){
```

Comment:
getTicket should be like regular getter
Create please field Ticket here and remove arguments from the method, also change return type